gen/src/index.ts

```diff
@@ -37,42 +33,52 @@ async function main() {
   const templateDirectoryPath = `${__dirname}/../../${TEMPLATE_DIRECTORY}`;
 
   const schemasAndGenFiles = await qt.getJSONSchemasAndGenFiles(IN, LANGUAGE);
-  const allEvents: Event[] = [];
+  const allEventsByPkg: Map<string, Event[]> = new Map<string, Event[]>();
   schemasAndGenFiles.map(([schema, genFile]: [any, string]) => {
     // Write generated Python scripts
     const pkg = schema['$id'];
     const pkgPath = pkg.replace(/\./g, '/');
     mkdirp.sync(`${OUT}/${SRC_DIRECTORY}/${pkgPath}`);
+    const eventName = schema.name;
     fs.writeFileSync(
-      `${OUT}/${SRC_DIRECTORY}/${pkgPath}/${PY_FILENAME}`,
+      `${OUT}/${SRC_DIRECTORY}/${pkgPath}/${eventName}.py`,
       genFile
     );
 
-    const pkgEvents: string[] = [];
-    Object.keys(schema.properties).map((eventName: string) => {
-      const eventDescription = schema.properties[eventName].description;
-      pkgEvents.push(eventName);
-      allEvents.push({
-        package: pkg,
-        eventName: eventName,
-        eventDescription: eventDescription,
-      });
-    });
+    const eventDescription = schema.description;
```

Comment:
Can you document this file a bit more.

Comments for every main section/block would be nice.